orchestrator/cosmos_gravity/src/send.rs

```diff
@@ -327,16 +348,57 @@ pub async fn send_ethereum_claims(
     }
 
     contact
-        .send_message(&msgs, None, &[fee], Some(TIMEOUT), private_key)
+        .send_message(&msgs, None, &[fee], Some(TIMEOUT), our_cosmos_key)
         .await
+        .map(Some)
+        .map_err(GravityError::CosmosGrpcError)
+}
+
+/// Creates the `Msg`s needed for `orchestrator` to attest to `events`
+/// If `eth_balances_by_block_height` is None, the `Msg`s will have empty `bridge_balances`,
+/// otherwise if no balances exist for any event's block_height, that event will be skipped
+/// Returns a Vec of (event_nonce: u64, Msg), which may be empty
+fn create_claim_msgs(
+    eth_balances_by_block_height: Option<HashMap<Uint256, Vec<ProtoErc20Token>>>,
```

Comment:
I think I have a conversion from protoerc20 to native erc20 rust type  you can use.